configs/prow-dev/jobs/ti-community-infra/test-dev/rustin-bot-periodics.yaml

```diff
@@ -111,4 +111,57 @@ periodics:
       volumes:
         - name: github-token
           secret:
+            secretName: github-token
+  - name: periodic-tichi-test-autobump
```

Comment:
By the way, these tasks are for the dev environment, so I suggest you add the tasks to configs after this repository has been tested with presubmits. One for tichi and one for configs.

Reply:
Is the .prow.yaml in the root directory of the tichi repo in dev environment or prod environment?

Reply:
yes